training/components.py

```diff
@@ -108,9 +108,16 @@ def pytorchjob_manifest_op(
     output_pvc_name: str,
     name_suffix: str,
     # path_to_model: str,
-    phase_name: str,
+    phase_num: int,
     nproc_per_node: int = 3,
     nnodes: int = 2,
+    num_epochs: int = 2,
+    effective_batch_size: int = 3840,
+    learning_rate: float = 1e-4,
+    num_warmup_steps: int = 800,
+    save_samples: int = 0,
+    max_batch_len: int = 20000,
+    seed: int = 42,
```

Comment:
WDYT? Should any of these be workflow params or is it fine to hardcode them into the component definition?

Reply:
`num_epochs` as a pipeline param would be very useful for tests with higher epochs. Having said that there are already quite some changes in this PR.

Reply:
I'm working on adding them now 😄  . The one thing is that this PR is required for training to run correctly in the pipeline now since the data has been split into a Skills and Knowledge directories. So it's probably best to merge this now and add the exposed params in a follow up. Unless folks are working on other stuff today and no one is blocked.

Reply:
Both is fine with me. It was an open question if it even makes any sense to expose them. 😄 A follow-up PR is good solution. I'm totally fine with that.